projects/opentrain/gtfs_pipeline/heatmap/HeatMap.java

```diff
@@ -0,0 +1,500 @@
+package projects.opentrain.gtfs_pipeline.heatmap;
+
+import com.projects.opentrain.gtfs_pipeline.Protos.Stop;
+import com.projects.opentrain.gtfs_pipeline.Protos.StopTime;
+import com.projects.opentrain.gtfs_pipeline.Protos.Trip;
+
+import com.google.startupos.common.flags.Flags;
+import com.google.startupos.common.flags.Flag;
+import com.google.startupos.common.flags.FlagDesc;
+
+import javax.imageio.ImageIO;
+import java.awt.Color;
+import java.awt.Graphics2D;
+import java.awt.Font;
+
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class HeatMap {
+
+  private List<StopsAround> stopsAroundList = new ArrayList<>();
+
+  private double[][] heatMap;
+  private List<Stop> stopsList = new ArrayList<>();
+  private List<StopTime> stopTimeList = new ArrayList<>();
+  private List<Trip> tripsList = new ArrayList<>();
+  private static final String tripsPath = "projects/opentrain/gtfs_pipeline/trips.protobin";
```

Comment:
`static finals` are constants, so should be `UPPER_CASE`. For example: `TRIPS_PATH`.